examples/mongodbatlas_encryption_at_rest/aws/atlas-cluster/README.md

```diff
@@ -70,15 +70,15 @@ terraform destroy
 
 1. Import the cluster using the Project ID and cluster name (e.g. `5beae24579358e0ae95492af-MyCluster`):
 
-        $ terraform import mongodbatlas_cluster.my_cluster ProjectId-ClusterName
+        $ terraform import mongodbatlas_advanced_cluster.my_cluster ProjectId-ClusterName
```

Comment:
Error from before (also on line 75, 78, and 82):
`my_cluster` -> `cluster`

Reply:
thx! changed